CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ Changelog
 
 Unreleased
 ----------
+### Changed
+* Unlocked the version of postcss from "v7.0.18" to "v7.0.21".
+
+5.11.0 - (October 25, 2019)
```

Comment:
ooops i forgot to push my 5.11 release changes into master. you can ignore those.